src/components/Form.js

```diff
@@ -1,15 +1,50 @@
-import React from 'react';
-
-const FormBook = () => (
-  <div>
-    <h2>Add New Book</h2>
-    <form>
-      <input type="text" placeholder="enter book name" />
-      <input type="text" placeholder="enter book author" />
-      <input type="text" placeholder="enter book percentage" />
-      <button type="submit">Add</button>
-    </form>
-  </div>
-);
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { v4 as uuid4 } from 'uuid';
+import { addBook } from '../redux/books/books';
```

Comment:
- [ ] Kindly this method comes from `'../redux/books/Books'` Books not books, to solve these errors.

![image](https://user-images.githubusercontent.com/35535130/163198764-3849013c-a5b8-4a63-b2ac-af19736e4ee0.png)